internal/examples/server/data/agent.go

```diff
@@ -106,6 +108,10 @@ func (agent *Agent) UpdateStatus(
 
 	agent.processStatusUpdate(statusMsg, response)
 
+	if statusMsg.ConnectionSettingsRequest != nil {
+		agent.processConnectionSettingsRequest(statusMsg.ConnectionSettingsRequest.Opamp, response)
```

Comment:
Yes, it possible. OpAMP spec does not require that the response is sent immediately in the following message. This can be processed asynchronously and a response may be sent "sometime in the future".